components/Card/Card.styled.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export const StyledCard = styled.article`
+  display: grid;
+  grid-template-columns: 40% 60%;
+  align-items: center;
+  justify-items: start;
+  padding: 1rem;
+  margin: 1rem;
+  border-radius: 1.5rem;
+  background-color: #f1d9a6;
```

Comment:
if you want to use this color multiple times, i would recommend to add a variable to your global styles and only use the variable in the components.